src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPage.java

```diff
@@ -339,14 +477,23 @@ private boolean checkIfPrivateMessagesNotAllowed(String userName) {
     return isElementOnPage(allowPrivateMassageButton);
   }
 
-  private WebElement getElementForUser(String userName, String selector) {
-    String userCss = String.format(selector, userName);
+  private WebElement getElementForUser(String userName) {
+    WebElement kk = (WebElement) By.cssSelector(String.format(PRIVATE_MESSAGE_USER_SELECTOR, userName));
     wait.forElementVisible(By.cssSelector(userCss));
     return driver.findElement(By.cssSelector(userCss));
   }
 
-  public String generateMessageFromUser(String userName) {
-    return String.format("Hello this is %s timestamp: ", userName) + getTimeStamp();
+  private WebElement userWebElement(String userName, String message) {
+    WebElement userOnPrivateSection = (WebElement) By.cssSelector(String.format(message, userName));
+    return userOnPrivateSection;
+  }
+
+  public boolean isUserInPrivateSectionDisplayed(String userName) {
+    try {
+      return userWebElement(userName, PRIVATE_MESSAGE_USER_SELECTOR).isDisplayed();
+    } catch (ElementNotFoundException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)